include/sys/fs/zfs.h

```diff
@@ -799,13 +799,15 @@ typedef struct pool_scan_stat {
 	uint64_t	pss_start_time;	/* scan start time */
 	uint64_t	pss_end_time;	/* scan end time */
 	uint64_t	pss_to_examine;	/* total bytes to scan */
-	uint64_t	pss_examined;	/* total examined bytes	*/
+	uint64_t	pss_examined;	/* total bytes located by scanner */
+	uint64_t	pss_issued;	/* total bytes checked by scanner */
```

Comment:
Yes, that is true. Is this really a big concern though? We require libzfs and libzfs_core to match the kernel version so those won't break. Are we aware of any external users that rely on these structs directly?